### PR TITLE
[Term Entry] Javascript Method : .parseFloat()

### LINKED ISSUE #4673: [Term Entry] Javascript Method : .parseFloat()
### Reason/inspiration (optional)

We would like a new term entry for JavaScript [Numbers Methods](https://www.codecademy.com/resources/docs/javascript/number-methods) about the `.parseFloat()` method. The entry should go in a new file in `docs/content/javascript/concepts/number-methods/terms/parseFloat/parseFloat.md.`

The entry should:

- Have an Introductory paragraph about what the method is and does.
- Have a syntax section illustrating the function in a   ` ```pseudo code` block.
- An example showing the function in a ` ```js` code block.
- A runnable codebyte example in a ` ```codebyte/javascript` block.

Please refer to the [term entry templates](https://github.com/Codecademy/docs/blob/main/documentation/term-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term Entry for Existing Concept Entry.

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [X] Issue is assigned

### PR DESCRIPTION
**Pull request by filming**
### Description

- Added the Number.parseFloat() term for Javascript

### Issue Solved
- Closes #4673 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

File: content/javascript/concepts/number-methods/terms/parseFloat/parseFloat.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+Note: JavaScript does not have the distinction of "floating point numbers" and "integers" on the language level.
+
+## Syntax
+
+```js
+Number.parseFloat(value);
+```
+
+- `value` (required): The value to parse, which is [coerced to a string](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String#string_coercion). Leading whitespaces in this value are ignored.
```
Comment (mamtawardhani) [inline content/javascript/concepts/number-methods/terms/parseFloat/parseFloat.md:27]:
```suggestion
- `value`: Refers to the input string that the function converts into a floating-point number.
```

File: content/javascript/concepts/number-methods/terms/parseFloat/parseFloat.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+## Examples
```
Comment (mamtawardhani) [inline content/javascript/concepts/number-methods/terms/parseFloat/parseFloat.md:29]:
```suggestion
## Example
```

File: content/javascript/concepts/number-methods/terms/parseFloat/parseFloat.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+Converting a value to a floating point number:
```
Comment (mamtawardhani) [inline content/javascript/concepts/number-methods/terms/parseFloat/parseFloat.md:31]:
```suggestion
The example below denotes the use of the `parseFloat` method to convert a value to a floating point number:
```

File: content/javascript/concepts/number-methods/terms/parseFloat/parseFloat.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+console.log(Number.parseFloat('  2.59  '));
+// Output: 2.59
+
+console.log(Number.parseFloat('1.49some non-digit characters'));
+// Output: 1.49
+
+console.log(Number.parseFloat('ABC32'));
+// Output: NaN
+```
+
```
Comment (mamtawardhani) [inline content/javascript/concepts/number-methods/terms/parseFloat/parseFloat.md:49]: Please use a ```shell block to show the output and remove the comments from the code above.

File: content/javascript/concepts/number-methods/terms/parseFloat/parseFloat.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+## Codebyte Example
+
+```codebyte/js
+# Before executing the code, what do you believe the result will be?
```
Comment (mamtawardhani) [inline content/javascript/concepts/number-methods/terms/parseFloat/parseFloat.md:53]:
```suggestion
```

**Review by mamtawardhani — CHANGES_REQUESTED** [review]
Hello @filming! Thank you for contributing to the Codecademy Docs! The entry is well written!

I've suggested a few changes in the entry, could you please review the suggestions given and modify the required parts at your earliest convenience? 
Thanks! 😄

**Comment by filming** [comment]
@mamtawardhani I have made all of the suggested/requested changes to the branch. Please feel free to let me know if there's anything else that could be updated after another review.

**Review by mamtawardhani — APPROVED** [review]
Thank you for making the required changes @filming! 😃 

Looks good for a second review!

**Review by letsbuildanything — APPROVED** [review]
Ready to be Merged! 🚀

**Pull request merged**